03-seeking-help.Rmd

```diff
@@ -110,6 +110,16 @@ your issue.
 
 > #### Challenge 1 {.challenge}
 > 
+> Look at the help for the `c` function. What do you expect will
+> happen if you evaluate the following: 
```

Comment:
Agreed, I made the change.

On Sun, Aug 23, 2015 at 9:27 PM, Naupaka Zimmerman <notifications@github.com
> wrote:

> In 03-seeking-help.Rmd
> <https://github.com/swcarpentry/r-novice-gapminder/pull/57#discussion_r37715460>
> :
>
> > @@ -110,6 +110,16 @@ your issue.
> >
> >  > #### Challenge 1 {.challenge}
> >  >
> > +> Look at the help for the `c` function. What do you expect will
> > +> happen if you evaluate the following:
>
> I think perhaps this question could be a bit more specific? Like: what
> type of vector do you think each of these will result in? Otherwise the
> type of answer you are looking for is ambiguous.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/r-novice-gapminder/pull/57/files#r37715460>
> .
>